package/harvester-os/files/etc/dracut.conf.d/50-cos-initrd.conf

```diff
@@ -0,0 +1,4 @@
+hostonly_cmdline="no"
+hostonly="no"
+compress="xz"
+add_dracutmodules+=" dmsquash-live "
```

Comment:
i will remove this since the elemental one also has `livenet`

Reply:
these we needed since the `harvester/os2` and upstream elemental images disable multipathd module in dracut config.

This is no longer needed as the changes have been made in `harvester/os2` as part of https://github.com/harvester/os2/pull/125